src/webextension/survey/components/fields/details.jsx

```diff
@@ -0,0 +1,15 @@
+import React, { Component } from 'react';
+
+export default class DetailsField extends Component {
+  render() {
+    const { input } = this.props;
+    return (
+      <div className="field">
+        <label htmlFor={input.name}>Any other comments?</label>
```

Comment:
Nit: Clicking these `<label>` fields didn't really work for me locally unless I modified the DOM in the dev tools to also add `id="foo"` in addition to the existing `name="foo"`.